docs/network/standard-exits.md

```diff
@@ -74,51 +88,67 @@ const childChain = new ChildChain({ watcherUrl });
 
 Every exit requires an extra fee called an [exit bond](/network/exitbonds) as an incentive mechanism for users of the OMG Network to exit honestly and challenge dishonest exits. The bond is currently fixed at an amount estimated to cover the gas cost of submitting a challenge. You can check the current gas cost at [ETH Gas Station](https://ethgasstation.info/).
 
-> The standard exit process is the same for both ETH and ERC20 UTXOs. The tutorial shows how to work with ERC20 tokens. For working with ETH, change `0xd74ef52053204c9887df4a0e921b1ae024f6fe31` value (ERC20 contract) into `OmgUtil.transaction.ETH_CURRENCY`.
+> The standard exit process is the same for both ETH and ERC20 UTXOs. The tutorial shows how to work with ERC20 tokens. For working with ETH, change `erc20Exit` into `ethExit`.
 
 ```js
 async function startStandardExit() {
   // check if the exit queue exists for a given token
-  const queueForTokenExists = await rootChain.hasToken(
-    "0xd74ef52053204c9887df4a0e921b1ae024f6fe31"
-  );
+  const queueForTokenExists = await rootChain.hasToken(erc20Exit.token);
   if (!queueForTokenExists) {
     // add the exit queue for this token if it doesn't exist
-    return rootChain.addToken({
-      token: "0xd74ef52053204c9887df4a0e921b1ae024f6fe31",
+    const addToken = await rootChain.addToken({
+      token: erc20Exit.token,
       txOptions: {
-        from: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-        privateKey:
-          "0xCD55F2A7C476306B27315C7986BC50BD81DB4130D4B5CFD49E3EAF9ED1EDE4F7",
+        from: erc20Exit.address,
+        privateKey: erc20Exit.privateKey,
+        gas: gaslLimit
       },
     });
   }
 
   // get utxo information
-  const alicesUtxos = await childChain.getUtxos(
-    "0x8CB0DE6206f459812525F2BA043b14155C2230C0"
+  const aliceUtxos = await childChain.getUtxos(erc20Exit.address);
+  const aliceUtxoToExit = aliceUtxos.find(
+    (i) => i.currency === erc20Exit.token
   );
-  const exitData = await childChain.getExitData(alicesUtxos);
+  const exitData = await childChain.getExitData(aliceUtxoToExit);
 
   // start a standard exit
-  rootChain.startStandardExit({
+  const receipt = await rootChain.startStandardExit({
     utxoPos: exitData.utxo_pos,
     outputTx: exitData.txbytes,
     inclusionProof: exitData.proof,
     txOptions: {
-      from: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-      privateKey:
-        "0xCD55F2A7C476306B27315C7986BC50BD81DB4130D4B5CFD49E3EAF9ED1EDE4F7",
+      from: erc20Exit.address,
+      privateKey: erc20Exit.privateKey,
+      gas: gaslLimit
     },
   });
+  console.log("Alice started a standard exit: " + receipt.transactionHash);
+
+  const exitId = await rootChain.getStandardExitId({
+    txBytes: exitData.txbytes,
+    utxoPos: exitData.utxo_pos,
+    isDeposit: aliceUtxoToExit.blknum % 1000 !== 0,
```

Comment:
this may not be clear to the reader as to why you are taking `aliceUtxoToExit.blknum % 1000 !== 0` as an input into this param?